features/fixtures/app/unhandled_scenario.go

```diff
@@ -22,7 +22,13 @@ func UnhandledCrashScenario(command Command) func() {
 func MultipleUnhandledErrorsScenario(command Command) func() {
 	scenarioFunc := func() {
 		//Make the order of the below predictable
-		notifier := bugsnag.New(bugsnag.Configuration{Synchronous: true})
+		notifier := bugsnag.New(bugsnag.Configuration{
```

Comment:
Suggestion: is it worth abstracting this out to a `ConfigureBaseNotifier` (or similar) function that does this? That would hopefully self-document and prevent future tests not having this set.

Reply:
I think it is being configured for every scenario. It's just for a few that I had to fix it was configured a second time.
This is the new expectation after this PR: "if you want to Configure a second time, you have to also provide custom endpoints again, otherwise default ones are used".
Where do you think we should document this?